impl/src/main/java/org/eclipse/parsson/JsonContext.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2013, 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
new file, no need to mark it as created in 2013

Reply:
ok :)